.github/workflows/test.yml

```diff
@@ -38,30 +38,30 @@ jobs:
         mkdir -p deps/jtreg
         mkdir -p /tmp/deps
 
-    - name: 'Download JDK 19'
-      id: download_jdk_19
+    - name: 'Download JDK 20'
+      id: download_jdk_20
       uses: oracle-actions/setup-java@v1.1.1
       with:
         website: jdk.java.net
-        release: 19
+        release: 20
         install: false
         
-    - name: 'Extract JDK 19'
+    - name: 'Extract JDK 20'
       shell: sh
       run: |
-        mkdir -p /tmp/deps/jdk-19
-        tar --strip-components=1 -xvf ${{ steps.download_jdk_19.outputs.archive }} -C /tmp/deps/jdk-19
-        ls -lah /tmp/deps/jdk-19
+        mkdir -p /tmp/deps/jdk-20
+        tar --strip-components=1 -xvf ${{ steps.download_jdk_20.outputs.archive }} -C /tmp/deps/jdk-20
+        ls -lah /tmp/deps/jdk-20
 
-    - name: 'Check Java 19 version'
+    - name: 'Check Java 20 version'
       shell: sh
       run: |
-        ${{ matrix.JAVA19_HOME }}/bin/java --version
+        ${{ matrix.JAVA20_HOME }}/bin/java --version
 
-    - name: 'Setup Java 18'
-      uses: oracle-actions/setup-java@v1.1.1
+    - name: 'Setup Java 17'
+      uses: oracle-actions/setup-java@v1.3.1
       with:
-        release: 18
+        release: 17
```

Comment:
Maybe we could add a comment here that says that this sets `JAVA_HOME` for gradle to run on?